### DISCUSSION TITLE
Migrating playground to typescript project

### DISCUSSION THREAD

**Discussion by adarshaacharya** [vercel/satori#212] [Ideas]
Since the current playground is currently on jsx, as the project grows bigger I think it will be better if we can convert it to typescript-based Nextjs project.

If we are okay with that would love to contribute and make PR on that.

**Comment by adarshaacharya** [comment]
@shuding any thoughts on this

**Reply by shuding** [reply]
Thanks for asking! I don't think that's a big priority for us at the moment (as we still have bugs to fix and the playground is just one page with a lot of hacks, not actually used by users). But if you have time, I'm happy to accept a PR too.

**Reply by adarshaacharya** [reply]
thank you, will do that :)
